src/phoenix/core/traces.py

```diff
@@ -294,8 +289,12 @@ def _process_span(self, span: pb.Span) -> None:
                 cumulative_attribute_name,
                 difference,
             )
+        # Update token count total
+        if existing_span:
+            self._token_count_total -= existing_span[LLM_TOKEN_COUNT_TOTAL] or 0
+        self._token_count_total += new_span[LLM_TOKEN_COUNT_TOTAL] or 0
         # Process previously orphaned spans, if any.
-        for orphan_span in self._orphan_spans[span_id]:
+        for orphan_span in self._orphan_spans.pop(span_id, ()):
             self._process_span(orphan_span)
```

Comment:
ideally these should be put into a separate queue (just for orphan spans), especially if the number orphan spans is large, but in practice, the number is probably small